packages/caliper-fabric/lib/configValidator.js

```diff
@@ -112,12 +116,25 @@ class ConfigValidator {
             }
         }
 
+        // validate organizationWallets section
+        if (config.organizationWallets) {
+            const organizations = Object.keys(config.organizationWallets);
+            for (const organization of organizations) {
```

Comment:
The existence of an org should be checked. The previous code block gathers the valid org names, so it's a simple check.

(It might be better to validate the `organizationWallets` section as a whole, so rules like "valid keys", and "at least one key" can be specified. But it's not a dealbreaker, the above check is what matters.)